README.org

```diff
@@ -32,6 +32,15 @@ Optionally you can install it via [[https://github.com/quelpa/quelpa][quelpa]] t
   (justl-recipe-width 25))
 #+end_example
 
+For [[https://github.com/doomemacs/doomemacs][doom emacs]], you can install as:
+
+#+begin_example elisp
+(use-package! justl
+  :config
+  (map! :n "e" 'justl-exec-recipe))
+  )
```

Comment:
I guess this line needs to be removed...

Reply:
of mapping "e" to execute in evil's normal mode?

Reply:
No, I mean there is an extra parenthesis at line no 41.